Specs/DateTime/French/DateTimeModel.json

```diff
@@ -95,13 +95,13 @@
     ]
   },
   {
-    "Input": "Je vais partir pour 3.5ans",
+    "Input": "Je vais partir pour 3,5ans",
```

Comment:
Please don't make such edits in specs. Previously existing cases that pass must continue to pass.
In some variants of French the decimal separator can also be ".".

Reply:
I can change it back, but it seems that the number parser will need to support this as well. Currently the number parser (used by the date parser on the substring "3.5" in the above case), returns "35" as the value for parsing "3.5".

Do you know if it is acceptable in French to use "." as a decimal separator _only_ within dates? Or with years specifically? I haven't had a chance to look at how the code for other programming languages handles this case. I will do this now.

Part of what motivated me to change it was that I noticed other test cases use a comma decimal separator. Notably `"je partirai pour 3,5 seconde "` on line 223 in DurationParser.json.

Reply:
I think I figured this out. There was a "mistranslation" into Java code in `BaseNumberParser`. More details at that location.